tests/integration/test_core.py

```diff
@@ -191,3 +191,66 @@ async def test_wordpress_theme_installation_error(
         assert (
                 unit.workload_status == ops.model.ActiveStatus.name
         ), "status should back to active after invalid theme removed from config"
+
+@pytest.mark.asyncio
+async def test_wordpress_install_uninstall_plugins(
+        ops_test: pytest_operator.plugin.OpsTest,
+        application_name,
+        unit_ip_list,
+        get_plugin_list_from_ip
+):
+    """
+    arrange: after WordPress charm has been deployed and db relation established
+    act: change plugins setting in config
+    assert: plugins should be installed and uninstalled accordingly
+    """
+    plugin_change_list = [
+        ["classic-editor", "classic-widgets"],
+        ["classic-editor"],
+        ["classic-widgets"],
+        []
+    ]
+    for plugins in plugin_change_list:
+        application = ops_test.model.applications[application_name]
+        await application.set_config({"plugins": ",".join(plugins)})
+        await ops_test.model.wait_for_idle()
+
+        for unit_ip in unit_ip_list:
+            expected_plugins = set(plugins)
```

Comment:
Minor comment, `plugin_change_list` could be defined as a list of sets since sets are iterable as well so `join` etc. should work, up to you

Reply:
Fixed, thanks!